ci/windows/build_common.psm1

```diff
@@ -0,0 +1,67 @@
+
+param ($CXX_STANDARD,
+       $GPU_ARCHS)
```

Comment:
```suggestion
Param(
    [Parameter(Mandatory)]
    [Alias("cxx")]
    [int]$CXX_STANDARD = 17,
    [Parameter(Mandatory)]
    [Alias("archs")]
    [string]$GPU_ARCHS = "70"
)

```

Reply:
I don't like positionals, they'll still work in this case, but I want to make sure they're also usable outside of that.